modules/nw-ovn-kubernetes-list-database-contents.adoc

```diff
@@ -0,0 +1,227 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-ovn-northbound-database-contents_{context}"]
+= Listing OVN northbound database contents with ovn-kubernetes
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (oc) installed.
+
+To understand logic flow rules you need to examine the northbound database and understand what objects are there to see how they are translated into logic flow rules.
+
+Follow this procedure to list OVN database contents with OVN-Kubernetes.
```

Comment:
This is not formatted according to prereq guidelines. Rewrite into an UL.

Reply:
i redesigned this section